README.md

```diff
@@ -63,7 +64,14 @@
 - **[View]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- LIVE DEMO -->
 
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://budget-app-4fhu.onrender.com/)
```

Comment:
- [ ] When I try to register or log in using the live demo, I got the following error, kindly consider fixing it
>![image](https://github.com/Tekhlay/Budget-app/assets/53402600/509746bc-7a7c-4e7e-89ae-2700aba4f4cd)